backend/tests/test_schwab_import.py

```
@@ -127,6 +127,131 @@ def test_fee_extraction_no_fees(self):
+            amount=1,
+            fees=fees,
+        )
+        # Inject the price field on the transferItem.
+        txn["transferItems"][0]["price"] = 0.30
+        result = map_schwab_transaction(txn)
+        assert result["premium"] == pytest.approx(0.30, abs=1e-4)
+
+    def test_premium_preserves_subpenny_precision(self):
+        """Gross-up math preserves sub-penny precision (e.g. $0.295).
+
+        Sell put @ $0.295 gross × 1 contract = $29.50 gross, minus $0.66 fees
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)